src/modules/Channel/components/MessageFeedbackModal/index.tsx

```diff
@@ -30,6 +30,11 @@ export default function MessageFeedbackModal(props: MessageFeedbackModalProps):
   const { stringSet } = useContext(LocalizationContext);
   const { isMobile } = useMediaQueryContext();
 
+  const [isEdit] = useState<boolean>(
+    message?.myFeedback
+    && selectedFeedback === message.myFeedback.rating,
+  );
+
```

Comment:
Is there a reason for using state?
Do we need to maintain the state as it was during the initial rendering?

Reply:
actually i wasn't sure to use state here. ill remove it since its not necessary

Reply:
done